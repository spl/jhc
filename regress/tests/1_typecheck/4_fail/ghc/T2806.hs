
{-# LANGUAGE MagicHash #-}
{-# OPTIONS_GHC -Werror #-}

-- Trac #2806

module Foo where

import GHC.Base

foo :: Int
foo = 3
    where (I# _x) = 4

