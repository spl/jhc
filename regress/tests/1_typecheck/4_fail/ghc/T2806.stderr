
T2806.hs:13:11:
    Warning: Pattern bindings containing unlifted types should use an outermost bang pattern:
               (I# _x) = 4
    In an equation for `foo':
        foo
          = 3
          where
              (I# _x) = 4

<no location info>: 
Failing due to -Werror.
